mmvb_backend/benchmarking_sessions/migrations/0002_auto_20200514_0933.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.0.4 on 2020-05-14 09:33
```

Comment:
This is just a general tip, don't worry about it for this PR.
When dealing with migrations we usually try to keep their numbers as low as possible, so if a a set of migrations haven't still been merged into the main branch and are still under a dev branch we usually try to [squash](https://docs.djangoproject.com/en/3.0/topics/migrations/#squashing-migrations) them. It can be a tricky process, though.